wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/templates/sample/Robot.java

```diff
@@ -130,9 +138,10 @@ public void autonomous() {
 	public void operatorControl() {
 		m_robotDrive.setSafetyEnabled(true);
 		while (isOperatorControl() && isEnabled()) {
-			// drive arcade style
+			// Drive arcade style
 			m_robotDrive.arcadeDrive(m_stick.getX(), m_stick.getY());
```

Comment:
X and Y arguments are reversed. Y should be inverted since joystick Y axis is negative forward.